plugin/cpg/src/main/java/org/opencds/cqf/ruler/cpg/dstu3/provider/CqlExecutionProvider.java

```diff
@@ -213,15 +213,15 @@ public Parameters evaluate(
 
 		if (prefetchData != null) {
 			return org.opencds.cqf.ruler.utility.dstu3.Parameters.newParameters(
-					org.opencds.cqf.ruler.utility.dstu3.Parameters.newPart(
-							"error", (OperationOutcome) evaluationHelper.createIssue(
-									"invalid parameters", "prefetchData is not yet supported")));
+					org.opencds.cqf.ruler.utility.dstu3.Parameters.newPart("invalid parameters",
```

Comment:
try adding:

import static org.opencds.cqf.ruler.utility.dstu3.Parameters.newParameters
import static org.opencds.cqf.ruler.utility.dstu3.Parameters.newPart

then remove the namespacing from the functions. The idea behind the utility functions is that they improve readability. If they don't do that it doesn't serve much purpose.